iamproperty/iamproperty.github.io
# Advanced tables examples

## Summary of Changes:
1. Example page to show how to do bespoke filtering

## Rationale
Demonstrate how bespoke table filtering can be done without breaking the built in table functions from the framework.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /examples/filter-by-age

## Ticket(s)
DS-44
DS-98
DS-97

## Checklist

## Notes
